packages/shared/validators.js

```diff
@@ -18,10 +21,17 @@ export function isDomSelector (selector: any) {
   } catch (error) {
     throwError('mount must be run in a browser environment like PhantomJS, jsdom or chrome')
   }
-
   try {
     document.querySelector(selector)
-    return true
+
+    const pseudoSelectors = ['.', '#', '[', ':', '>', ' ']
```

Comment:
Quite ugly but needed to cover all existing test cases:

- `find("#id")`
- `find(".class")`
- `find("div")` html element
- `find("text")` svg element
- `find("div > p") pseudo selector, although this is prob bad practise in tests anyway
- `find("p:first-of-type")`